src/generate/generators/ExtensionFull.ts

```diff
@@ -92,13 +92,17 @@ export class AzExtensionFullGenerator extends GeneratorBase {
 
         await this.generateFullSingleAndAddtoOutput(new CliTestInit(this.model));
         await this.generateFullSingleAndAddtoOutput(new CliTestStep(this.model), true, true);
+        const hasTestResourceScenario = this.model.GetHandler().exampleHandler.GetResourcePool()
+            .hasTestResourceScenario;
         for (const testGroup of exampleHandler.Example_TestScenario
             ? Object.getOwnPropertyNames(exampleHandler.Example_TestScenario)
             : []) {
             await this.generateFullSingleAndAddtoOutput(
                 new CliTestScenario(
                     this.model,
-                    PathConstants.fullTestSceanrioFile(testGroup),
+                    hasTestResourceScenario
+                        ? PathConstants.testSwaggerScenarioFile(testGroup)
+                        : PathConstants.fullTestSceanrioFile(testGroup),
```

Comment:
we are not going to merge these four generators because of the transition ?

Reply:
currently don't have time to do this refactor